Sources/Terminal/AngouriMath.Terminal.Lib/AssemblyLoadBuilder.fs

```diff
@@ -0,0 +1,41 @@
+﻿module AngouriMath.Terminal.Lib.AssemblyLoadBuilder
+
+open System
+open Microsoft.DotNet.Interactive.FSharp
+open AngouriMath.Terminal.Lib.Consts
+
+type AssemblyLoadResult =
+    | AssemblyLoadSuccess
+    | AssemblyLoadFailure of string
+
+type AssembliesLoadResult =
```

Comment:
This is just `Result<FSharpKernal, string list>`

Reply:
Thanks